client/src/components/Breadcrumbs.jsx

```diff
@@ -0,0 +1,28 @@
+import './Breadcrumbs.css'
+
+const Breadcrumbs = ({ currentPath, fetchFiles }) => {
+  return (
+    <nav aria-label='Breadcrumb' className='breadcrumbs'>
+      <ul className='breadcrumb'>
+        <li className='breadcrumb-item' onClick={() => fetchFiles('/')}>
+          home
+        </li>
+        {currentPath !== '/' &&
+          currentPath
+            .split('/')
+            .slice(1)
+            .map((name, index, array) => {
+              const path = `/${array.slice(0, index + 1).join('/')}`
+
+              return (
+                <li key={index} className='breadcrumb-item' onClick={() => fetchFiles(path)}>
+                  <span>{name}</span>
+                </li>
+              )
+            })}
+      </ul>
+    </nav>
+  )
+}
+
+export default Breadcrumbs

```

Comment:
awesome! we are almost there! :) 
now let's do couple more cool things, shall we? 

what we need to do now is: 
1. externalise the logic inside `<ul>` element: that creates `<li>` with `home` and does `currentPath.split('/').slice(1)` - into a function,  name that function as `pathToBreadcrumbs()`
2. implement pathToBreadcrumbs function so that it worked as this: (input --> output)
  a.  "/" --> ```[ {name: "home", path: "/"} ]```
  b. "/one" --> ``` [ {name: "home", path: "/"}, {name: "one", path: "/one"} ]```
  c. "/one/two" -->  ```[ {name: "home", path: "/"}, {name: "one", path: "/one"}, {name: "two", path: "/one/two"} ]```
  d. and so on and so forth....
3. render li elements out of that array of objects, you'd need to use Array.map() method again.

so, the main idea is to have return as simple as possible.
there should not be any business logic or complex transformation done inside render (return) clause of a React component.